src/main/java/io/github/thebusybiscuit/slimefun4/api/ErrorReport.java

```diff
@@ -26,10 +28,15 @@
 import me.mrCookieSlime.Slimefun.api.Slimefun;
 
 public class ErrorReport {
-	
+
+	private static final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
```

Comment:
Date Formats are afaik not thread safe and shouldn't be static, should they...?

Reply:
Not the biggest fan of it having to be initialized every time... performance wise it makes me want to cry. Not sure there'd ever be much of a case where it'd have an issue thread wise but we can synchronize the use anyway if need be.